src/librawspeed/decompressors/FujiDecompressor.cpp

```diff
@@ -302,7 +302,8 @@ void FujiDecompressor::fuji_decode_sample(
   }
 
   if (code < 0 || code >= common_info.total_values) {
-    ThrowRDE("fuji_decode_sample");
+    ThrowRDE("fuji_decode_sample. code = '%d', total_values = '%d'", code,
```

Comment:
Major nitpick: i think it's generally useless to be verbose in these error bailouts.
You won't ever be able to print all necessary information,
but you could easily get that information by reproducing locally in a debugger.